src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryAdder.java

```diff
@@ -53,7 +53,7 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.util.regex.Pattern;
 import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
better to replace https://github.com/jenkinsci/pipeline-groovy-lib-plugin/blob/56393caa61b2107792d22f2721e9bda4249eaff9/src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryAdder.java#L231